content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Troubleshooting: Longhorn RWX shared mount ownership is shown as nobody in consumer Pod"
+author: Chin-Ya Huang
+draft: false
+date: 2021-03-31
+categories:
+  - "rwx"
+---
+
+## Applicable versions
+
+All Longhorn versions.
```

Comment:
Actually just 1.1.0 which starts having RWX support. Please double confirm.

Reply:
Yes you are right, https://github.com/longhorn/longhorn/releases/tag/v1.1.0.